guides/common/modules/proc_provisioning-ubuntu-20-04-3-plus-through-smart-proxies.adoc

```diff
@@ -0,0 +1,36 @@
+[id="Provisioning_Ubuntu_20_04_3_plus_Through_Smart_Proxies_{context}"]
+= Provisioning Ubuntu 20.04.3+ Through {SmartProxies}
```

Comment:
Do you like this title @bastian-src ? Should it rather include "Ubuntu Autoinstall"?

Reply:
Yes, we should say:
```suggestion
= Provisioning Ubuntu Autoinstall Through {SmartProxies}
```
Since you mention the specific versions in the short notice beneath, we don't need the version here.

Reply:
Applied.